make/configure.py

```diff
@@ -1891,6 +1891,10 @@ class Tools:
             doc.add( 'HAS.regex', 1 )
         if strtok_r.fail:
             doc.add( 'COMPAT.strtok_r', 1 )
+
```

Comment:
OK assuming the strerror_r check is unnecessary for freebsd

Reply:
oh no, it's necessary.
I will fix this.

Reply:
c20d42d84aaa12f1d9ad715eccdf723c2083e655 commit works on my FreeBSD.

build/log/config.info.txt shows this message.
```
probe: strerror_r...(pass) end
```

How about this?